config/main.py

```diff
@@ -1591,6 +1591,79 @@ def incremental(file_name):
     command = "acl-loader update incremental {}".format(file_name)
     run_command(command)
 
+
+#
+# 'dropcounter' group ('config dropcounter ...')
+#
+
+@config.group()
+def dropcounter():
+    """Drop counter related configuration tasks"""
+    pass
+
+
+#
+# 'initialize_counter' subcommand ('config dropcounter initialize_counter')
+#
+@dropcounter.command()
+@click.argument("counter_name", type=str, required=True)
+@click.argument("counter_type", type=str, required=True)
+@click.argument("reasons",      type=str, required=True)
+@click.option("-a", "--alias", type=str, help="Alias for this counter")
+@click.option("-g", "--group", type=str, help="Group for this counter")
+@click.option("-d", "--desc",  type=str, help="Description for this counter")
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def initialize_counter(counter_name, alias, group, counter_type, desc, reasons, verbose):
```

Comment:
Is there a reason to include the suffix "_counter" here? It seems a bit redundant. Is it to distinguish it from the "_reasons" subcommands commands below?